src/App.test.js

```diff
@@ -1,8 +1,60 @@
-import { render, screen } from "@testing-library/react";
+import { render, screen, waitFor } from "@testing-library/react";
 import App from "./App";
+import * as api from "./api";
 
-test("renders title", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/2023 React Fundamentals Workshop/i);
-  expect(linkElement).toBeInTheDocument();
+const jobTitles = ["Accountant", "Engineer", "Lawyer", "Teacher"];
+
+describe("<App />", () => {
+  const getJobTitles = jest.spyOn(api, "getJobTitles");
+  const addAttendant = jest.spyOn(api, "addAttendant");
+  const getAttendants = jest.spyOn(api, "getAttendants");
+
+  beforeEach(() => {
+    getJobTitles.mockResolvedValue({
+      status: 200,
+      data: jobTitles,
+    });
+    addAttendant.mockResolvedValue({
+      status: 200,
+    });
+    getAttendants.mockResolvedValue({
+      status: 200,
+      data: [],
+    });
+  });
+
+  it("renders component correctly on successful initial load", async () => {
```

Comment:
Company practice is to use `when`
```suggestion
  it("renders component correctly when data is loaded", async () => {
```